tdms/src/tdms_iterator.cpp

```diff
@@ -5940,28 +5940,31 @@ void mexFunction(int nlhs, mxArray *plhs[], int nrhs, const mxArray *prhs[])
     if(  (((double)time(NULL)) - t0)>1 ){
  
       maxfield = 0.;
-      for(k=0;k<(K_tot+1);k++)
-	for(j=0;j<(J_tot+1);j++)
-	  for(i=0;i<(I_tot+1);i++){
-	    tempfield = fabs(Exy[k][j][i]+Exz[k][j][i]);
-	    if( maxfield < tempfield )
-	      maxfield = tempfield;
-	    tempfield = fabs(Eyx[k][j][i]+Eyz[k][j][i]);
-	    if( maxfield < tempfield )
-	      maxfield = tempfield;
-	    tempfield = fabs(Ezx[k][j][i]+Ezy[k][j][i]);	
-	    if( maxfield < tempfield )
-	      maxfield = tempfield;
-	    tempfield = fabs(Hxy[k][j][i]+Hxz[k][j][i]);	
-	    if( maxfield < tempfield )
-	      maxfield = tempfield;
-	    tempfield = fabs(Hyx[k][j][i]+Hyz[k][j][i]);	
-	    if( maxfield < tempfield )
-	      maxfield = tempfield;
-	    tempfield = fabs(Hzx[k][j][i]+Hzy[k][j][i]);	
-	    if( maxfield < tempfield )
-	      maxfield = tempfield;
-	  }
+      for(k=0;k<(K_tot+1);k++) {
+        for (j = 0; j < (J_tot + 1); j++){
+          for (i = 0; i < (I_tot + 1); i++) {
+            tempfield = fabs(Exy[k][j][i] + Exz[k][j][i]);
+            if (maxfield < tempfield)

```

Comment:
Since you're modifying also this block, could add the brackets also to these `if`s